README.md

```diff
@@ -141,15 +141,15 @@ git clone https://github.com/kubernetes-client/gen
 
 ```bash
 # Where REPO_DIR points to the root of the csharp repository
-cd 
+cd
 ${GEN_DIR}/openapi/csharp.sh ${REPO_DIR}/src/KubernetesClient ${REPO_DIR}/csharp.settings
 ```
 
-# Version Compatibility 
+# Version Compatibility
 
 | SDK Version | Kubernetes Version | .NET Targeting                        |
 |-------------|--------------------|---------------------------------------|
-| 7.0         | 1.23               | netstandard2.1;net5;net6              |
+| 7.0         | 1.23               | netstandard2.1;net5.0;net6.0          |
 | 6.0         | 1.22               | netstandard2.1;net5                   |
```

Comment:
```suggestion
| 6.0         | 1.22               | netstandard2.1;net5.0                 |
```

Reply:
This PR will pertain only to 7.x packages, unless we port it to previous version, this I deliberately not modified this particular value.

Reply:
But OK, I'll incorporate your suggestion.